### PR TITLE
V7 features

### PR DESCRIPTION
**Pull request by EhsanKia**
- Port Better Viewer List to V7
- Wrap module loading in try/catch
- Optimize name color calculation and refactor to colors module
- Move dark-css module inside global-css for ordering

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 (() => {
     if (window.location.pathname.endsWith('.html')) return;
-
     const debug = require('./utils/debug');
-    require('./modules/**/index.js', {mode: 'expand'});
+
+    require('./modules/**/index.js', {mode: function(base, files) {
+        return files.map((module) => {
+            return `try { require('${module}') } catch (e) { debug.log('Failed to load ${module}\\n', e); }`;
```
Comment (night) [inline src/index.js]:
```
return `
    try {
        require('${module}');
    } catch (e) {
        debug.log('Failed to load module "${module}"', e);
    }
`;
```

this seems more readable, and you shouldn't have to join by newline?

File: src/modules/better_viewer_list/templates.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+var button = `
```
Comment (night) [inline src/modules/better_viewer_list/templates.js]: const

File: src/modules/better_viewer_list/templates.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<a id="bvl-button" title="Better Viewer List" class="button button--icon-only float-left">
+  <figure>
+    <svg version="1.1" viewbox="0 0 16 16" width="16px" height="16px" class="svg-betterviewerlist">
+      <path clip-rule="evenodd" d="M6,15v-2h8v2H6z M6,9h8v2H6V7z M6,5h8v2H6V3z M6,1h8v2H6V3z M2,13h2v2H2V11z M2,9h2v2H2V7z M2,5h2v2H2V3z M2,1h2v2H2V11z"></path>
+    </svg>
+  </figure>
+</a>`;
+
+var panel = `
```
Comment (night) [inline src/modules/better_viewer_list/templates.js]: const

File: src/modules/better_viewer_list/index.js [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+            return;
+        }
+
+        var interval = setInterval(() => {
+            if ($('#bvl-button').length > 0) {
+                clearInterval(interval);
+                return;
+            }
+
+            var $oldViewerList = $('.chat-buttons-container a.button:has(.svg-viewerlist)');
```
Comment (night) [inline src/modules/better_viewer_list/index.js]: const

File: src/modules/better_viewer_list/index.js [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+                    this.createPanel();
+                }
+            });
+
+            clearInterval(interval);
+        }, 100);
+    }
+
+    renderViewerList() {
+        var count = 0;
```
Comment (night) [inline src/modules/better_viewer_list/index.js]: let

File: src/modules/better_viewer_list/index.js [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+        var results = chatterList;
```
Comment (night) [inline src/modules/better_viewer_list/index.js]: ..etc

**Comment by EhsanKia** [comment]
Fixed

**Pull request merged**